.github/workflows/_publish_to_pypi.yaml

```diff
@@ -19,7 +25,7 @@ jobs:
       id-token: write # Required for OIDC authentication.
     environment:
       name: pypi
-      url: https://pypi.org/project/crawlee
+      url: https://pypi.org/project/${{ inputs.package_name }}
 
     steps:
       - name: Checkout repository
```

Comment:
It should be possible to configure the ref for this one, so that we have an up-to-date changelog

Reply:
I would say so.